vindex/api/api.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2025 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// api contains the API definitions for the prototype VIndex API.
+package api
+
+import "crypto/sha256"
+
+const (
+	// PathLookup defines the path from the vindex base URL where the lookup
+	// operation will be served. This takes a single key as a GET request
+	// parameter, and returns a marshalled LookupResponse.
```

Comment:
Is this comment out of date? From the client code, it looks like it actually is used as a path prefix, and is followed by a hex-encoded SHA256 of the lookup key?